libs/talent-pool/config/src/lib/talent-pool-config.ts

```diff
@@ -760,7 +760,8 @@ export const germanFederalStates = {
   'rheinland-pfalz': 'Rheinland-Pfalz',
   saarland: 'Saarland',
   sachsen: 'Sachsen',
-  sachsenAnhalt: 'Sachsen-Anhalt',
-  schleswigHolstein: 'Schleswig-Holstein',
+  'sachsen-anhalt': 'Sachsen-Anhalt',
```

Comment:
Do you plan to run a `db.update()` for any existing job listings with the old values?

Reply:
Yes if there is any. I don't think we have any though or even if we have, I can easily update them.

Reply:
👍